CHAOSScon/2020EU/schedule.md

```diff
@@ -1,3 +1,305 @@
+<td align=left><small><!--Session-->
+<b>Measuring Culture</b><br/>
+<cite><author>Sharan F.</author></cite>
+<td align=center><small>
+
+[<kbd>Slides</kbd>]()
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 11:00 - 11:20 --><!--CONFIRMED-->
+<tr valign=top>
+<th align=right><time>11:00</time>
+<td align=left><small><!--Session-->
+<b>Technical Lag: Measuring the outdatedness, vulnerability and bugginess of software</b><br/>
+<cite><author>Tom Mens</author></cite>
+<td align=center><small>
+
+[<kbd>Slides</kbd>]()
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 11:20 - 11:50 --><!--CONFIRMED-->
+<tr valign=top>
+<th align=right><time>11:20</time>
+<td align=left><small><!--Session-->
+<b>CHA-IO-SS - Community Health Analytics for Inner and Open Source Software</b><br/>
+<cite><author>Daniel I.</author></cite>
+<td align=center><small>
+
+[<kbd>Slides</kbd>]()
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<tbody>
+
+<!-- 11:50 - 13:00 --><!--CONFIRMED--><!--INTERIM-->
+<tr valign=top>
+<th align=right><time>11:50</time>
+<th colspan=2><!--Interim-->
+Lunch & Networking
+
+<!-- 13:00 - 13:20 --><!--UNCONFIRMED--><!--KEYNOTE-->
+<tr valign=top>
+<th align=right><time>13:00</time>
+<td align=center colspan=2><small><!--Session-->
+<b><samp style="color:#9999">TBA</samp></b><br/>
+<cite><author hidden><samp style="color:#9999">TBA</samp></author></cite>
+<td hidden align=left><small><!--Session-->
+<td hidden align=center><small>
+
+<!-- [<kbd>Slides</kbd>]() -->
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 13:20 - 13:40 --><!--UNCONFIRMED--><!--SESSION-->
+<tr valign=top>
+<th align=right><time>13:20</time>
+<td align=center colspan=2><small><!--Session-->
+<b><samp style="color:#9999">TBA</samp></b><br/>
+<cite><author hidden><samp style="color:#9999">TBA</samp></author></cite>
+<td hidden align=left><small><!--Session-->
+<td hidden align=center><small>
+
+<!-- [<kbd>Slides</kbd>]() -->
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 13:40 - 14:00 --><!--UNCONFIRMED-->
+<tr valign=top>
+<th align=right><time>13:40</time>
+<td align=center colspan=2><small><!--Session-->
+<b><samp style="color:#9999">TBA</samp></b><br/>
+<cite><author hidden><samp style="color:#9999">TBA</samp></author></cite>
+<td hidden align=left><small><!--Session-->
+<td hidden align=center><small>
+
+<!-- [<kbd>Slides</kbd>]() -->
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 14:00 - 14:20 --><!--CONFIRMED-->
+<tr valign=top>
+<th align=right><time>14:00</time>
+<td align=left><small><!--Session-->
+<b>Navigating Community Health Through Comparison</b><br/>
+<cite><author>Kevin L.</author></cite>
+<td align=center><small>
+
+[<kbd>Slides</kbd>]()
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 14:20 - 14:50  --><!--UNCONFIRMED--><!--SESSION-->
+<tr valign=top>
+<th rowspan=4><time>14:20</time>
+<th colspan=2><!--Session-->
+Lightning Talks
+
+<!-- 14:20 - 14:50  --><!--CONFIRMED-->
+<tr valign=top><td><small><b>Metrics and valuation of open source projects:</b><br/> A crossover between Open Source and Open Science
```

Comment:
line 167, please remove the linebreak from the title and add the speaker name(s).